lib/BloqadeQMC/src/operatorsamplers/operatorsampler.jl

```diff
@@ -0,0 +1,28 @@
+abstract type AbstractOperatorSampler{K, T, P <: AbstractProbabilityVector{T}} end
```

Comment:
do we actually need this abstract type?

Reply:
Probably not, it's there because I cut out another OpSampler that isn't used much (might need it later though for large systems)